adoc/MAIN.release-notes.adoc

```diff
@@ -19,6 +19,26 @@ Release notes usually only list changes that happened between two subsequent rel
 Certain important entries from the release notes documents of previous product versions may be repeated.
 To make such entries easier to identify, they contain a note to that effect.
 
+== Changes in 4.3.0
+
+=== {kube} 1.18
+
+FILLIN
+
+=== Required Actions
+
+=== {crio}
+
+This release upgrades `cri-o` to 1.18, which brings support for {kube} 1.18. This new version, though, requires some manual steps for the upgrade, since the configuration files have been migrated from `sysconfig` to `/etc/crio/crio.conf.d/`. In order to upgrade, then, you will have to perform the following commands:
```

Comment:
```suggestion
This release upgrades `cri-o` to 1.18, which brings support for {kube} 1.18. This new version, though, requires some manual steps for the upgrade, since the configuration files have been migrated from `sysconfig` to `/etc/crio/crio.conf.d/01-caasp.conf`. In order to upgrade, then, you will have to perform the following commands:
```